app/react/App/scss/modules/_metadata-view.scss

```diff
@@ -50,6 +50,19 @@
           content: ',';
         }
       }
+
+      button.link-button {
+        background: none !important;
+        border: none;
+        padding: 0 !important;
+        color: #2b56c1;
```

Comment:
I believe we have some sort of variables for link colors?

Reply:
Found